barretenberg/cpp/bootstrap_cache.sh

```diff
@@ -5,13 +5,20 @@ cd "$(dirname "$0")"
 source ../../build-system/scripts/setup_env '' '' mainframe_$USER > /dev/null
 
 echo -e "\033[1mRetrieving bb binary from remote cache...\033[0m"
-extract_repo_if_working_copy_clean barretenberg-x86_64-linux-clang \
-  /usr/src/barretenberg/cpp/build/bin ./build \
-  /usr/src/barretenberg/cpp/build-pic/lib ./build-pic
+HASH=$(AZTEC_CACHE_REBUILD_PATTERNS=.rebuild_patterns compute-content-hash.sh)
+TMP=$(mktemp -d)
 
-echo -e "\033[1mRetrieving bb.wasm from remote cache...\033[0m"
-extract_repo_if_working_copy_clean barretenberg-wasm-linux-clang \
-  /usr/src/barretenberg/cpp/build-wasm/bin ./build-wasm \
-  /usr/src/barretenberg/cpp/build-wasm-threads/bin ./build-wasm-threads
+function on_exit() {
+  rm -rf "$TMP"
+}
+trap on_exit EXIT
 
-remove_old_images barretenberg-wasm-linux-clang
+cache-download.sh barretenberg-preset-wasm-$HASH.tar.gz $TMP/build-wasm > /dev/null
+cache-download.sh barretenberg-preset-wasm-threads-v1-$HASH.tar.gz $TMP/build-wasm-threads > /dev/null
```

Comment:
you can change the -v1 back to no suffix, that was a hack to not have to make a spurious edit to bb because I was changing the earthly line

Reply:
(obviously change it everywhere)